src/App/Pages/Generator/GeneratorPage.xaml.cs

```diff
@@ -61,25 +61,31 @@ public async Task InitAsync()
         protected async override void OnAppearing()
         {
             base.OnAppearing();
+
+            _vm?.RedrawPassword(); // this is needed when coming back from another view to this one and the theme has changed
```

Comment:
it shouldn't be, at least not on all cases I've tested, but I don't want the app to crash because we didn't consider some scenario. At the top of my head, it should never be null